pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+version = "v1.1.0"
+name = "candigv2_authx"
+dependencies = [
```

Comment:
Q: Does Snyk check these deps?

Reply:
It's a little unclear to me, based on https://docs.snyk.io/scan-using-snyk/supported-languages-and-frameworks/python. I'm not sure it was checking setup.py before either, unless we'd told it to do that.